src/sqlancer/stonedb/gen/StoneDBInsertGenerator.java

```diff
@@ -45,9 +45,25 @@ private SQLQueryAdapter getQuery() {
         sb.append(table.getName());
         appendPartition();
         appendColumnsAndValues(columns);
+        addExpectedErrors();
         return new SQLQueryAdapter(sb.toString(), errors);
     }
 
+    private void addExpectedErrors() {
```

Comment:
I just noticed that `private final StringBuilder sb = new StringBuilder();` in this class is redundant, since the field is already defined in the superclass.